insecure/corruptible/factory.go

```diff
@@ -238,8 +240,13 @@ func (c *ConduitFactory) processAttackerMessage(msg *insecure.Message) error {
 		return fmt.Errorf("could not convert target ids from byte to identifiers: %w", err)
 	}
 
-	lg = lg.With().Str("target_ids", fmt.Sprintf("%v", msg.TargetIDs)).Logger()
-	err = c.sendOnNetwork(event, network.Channel(msg.ChannelID), msg.Protocol, uint(msg.TargetNum), targetIds...)
+	lg = lg.With().
+		Str("target_ids", fmt.Sprintf("%v", targetIds)).
+		Uint32("targets_num", msg.TargetNum).
```

Comment:
was this change intentional? It seems unrelated to the slashing hooks

Reply:
This is result of fixing merge conflict due to updating the channels import

Reply:
this diff is against master, so just checking that you are intending to add this label that doesn't exist on master